package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lib-oauth-tooling",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
As TokenInfo is obsolete and might be used by consuming programs, version should switch to 3.0.0